frontend/src/concepts/dashboard/DashboardPopupIconButton.tsx

```diff
@@ -13,7 +13,7 @@ const DashboardPopupIconButton = ({
   ...props
 }: DashboardPopupIconButtonProps): React.JSX.Element => (
   <Button variant="plain" isInline style={{ padding: 0 }} {...props}>
-    <Icon tabIndex={0} isInline style={{ marginLeft: 'var(--pf-v5-global--spacer--xs)' }}>
+    <Icon isInline style={{ marginLeft: 'var(--pf-v5-global--spacer--xs)' }}>
```

Comment:
@andrewballantyne that's correct. The `<Icon>` component didn't need/shouldn't be focusable itself as it causes extraneous Tabbing (not super huge at first glance, but with many of the help popover buttons on a page it adds up) and the `<Button>` component that wraps it will already receive focus. The end result remains the same, though, i.e. the help popover button receives focus and will trigger the popover open on click.